src/index.js

```diff
@@ -24,15 +28,10 @@ async function start(fields) {
   await authenticate(fields.login, fields.password)
   log('info', 'Successfully logged in')
 
-  log('info', 'Fetching bills information')
-  const bills = await fetchAndParseBills()
-  log('info', 'Fetched bills information')
-
-  log('info', 'Downloading bills')
-  await saveFiles(bills, fields.folderPath)
+  await handleBills(fields)
 }
 
-async function fetchAndParseBills() {
+async function handleBills(fields) {
```

Comment:
The name of this function is a smell: what does `handleBills` handle?

As far as I can see, it:
- constructs a `bill` object
  - taking data from the webpage
  - taking other data from a pdf 
- assembles an array
- save every array item into cozy stack with `saveBills`

I guess you can divide the workflow into simple function with one responsibility (SRP). Then the name of the function will not be enigmatic or vague.